### DISCUSSION TITLE
Json Vehicles and outfit

### DISCUSSION THREAD

**Discussion by Shenmuee** [YimMenu/YimMenu#2385] [General]
here is some of my json vehicles if you can share yours here that would be nice. You can share your outfits too.
[saved_json_vehicles.zip](https://github.com/YimMenu/YimMenu/files/13258137/saved_json_vehicles.zip)

**Comment by a7cATR4z** [comment]
Where do I put the folder?

**Comment by xesdoog** [comment]
[json_vehicles.zip](https://github.com/user-attachments/files/16827139/json.zip)
[xml_vehicles.zip](https://github.com/user-attachments/files/16827140/xml_vehicles.zip)

**Comment by MindOfBog** [comment]
I have compiled all 12 alien outfits available on GTAO, all of them with boots matching their color so that they aren't bare foot :)

They can ALL be saved as outfits and used without a mod menu!

[all alien outfits.zip](https://github.com/user-attachments/files/16886996/all.alien.outfits.zip)

![image_2024-09-05_122424829](https://github.com/user-attachments/assets/b906a07c-573a-4688-a811-267e0a36648a)

**Reply by MindOfBog** [reply]
P.S. if you don't like the boots you can take them off without the outfit getting messed up
